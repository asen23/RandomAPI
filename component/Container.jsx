export default function Container(props) {
    return (
        <div className="flex justify-center items-center flex-col min-h-screen gap-8 bg-[antiquewhite]">
            {props.children}
        </div>
    )
}
