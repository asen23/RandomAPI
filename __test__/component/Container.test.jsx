import { render, screen } from "@testing-library/react"
import userEvent from "@testing-library/user-event"
import Container from "component/Container"

describe("Container", () => {
    it("renders unchanged", async () => {
        const { asFragment } = render(<Container>A Component</Container>)
        expect(asFragment()).toMatchSnapshot()
    })

    it("renders its children", () => {
        render(<Container>A Component</Container>)

        const child = screen.getByText("A Component")
        expect(child).toBeInTheDocument()
    })
})
