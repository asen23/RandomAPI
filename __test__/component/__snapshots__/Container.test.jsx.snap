// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container renders unchanged 1`] = `
<DocumentFragment>
  <div
    class="flex justify-center items-center flex-col min-h-screen gap-8 bg-primary"
  >
    A Component
  </div>
</DocumentFragment>
`;
